src/index.js

```diff
@@ -1,9 +1,9 @@
 'use strict';
 
 const app = require('./app');
-const port = app.get('port');
+const port = process.env.PORT || app.get('port');
```

Comment:
Does this not already get pulled in by feathers-configuration if you have `PORT` set as an environment var?